### PR TITLE
Bug: cannot analyse value if is interpolated

### PR DESCRIPTION
**Pull request by Gert-JanPeeters**
I ran into the same issue as described in #21 and #28. 

This also supports `blade` and `php`, next to `erb`.

ToDo:
- [x] data-controller
- [x] data-[identifier]-[value]-value
- [x] data-[identifier]-target
- [x] data-action
- [x] cleanup

### CODE REVIEW

File: server/src/diagnostics.ts [hunk 1]
```diff
@@ -95,7 +95,12 @@ export class Diagnostics {
       const value = attributeValue(node, attribute) || ""
       const attributeMatches = attribute.match(this.valueAttribute)
 
-      // TODO: skip when value contains <%= or %>
+      // cannot analyze value if it is interpolated       
+      const skippableTags = ["<%=", "<%-", "%>", "<?=", "<?php", "?>", "{{", "}}"]
```
Comment (marcoroth) [inline server/src/diagnostics.ts:99]: Is there a reason you left out `<%`?

File: server/src/diagnostics.ts [hunk 2]
```diff
@@ -95,7 +95,12 @@ export class Diagnostics {
...
+      const foundSkippableTags = skippableTags.filter((tag) => value.includes(tag)).length
```
Comment (marcoroth) [inline server/src/diagnostics.ts:100]: I feel like this reads easier:
```suggestion
      const foundSkippableTags = skippableTags.some((tag) => value.includes(tag))
```

**Review by marcoroth — COMMENTED** [review]
Thanks for working on this @Gert-JanPeeters! I think this resolves the issues at hand.

Though, there are some cases we should think about. I'm not sure if this should also happen in this PR, but here it goes:

* Currently this only fixes it for `data-[identifier]-[value]-value` attributes, but technically this could be the case for any Stimulus-attribute. 

  So maybe it would make sense to extract this logic out of the `validateDataValueAttribute()` function and apply it for all the data attribute validation methods in the `diagnostics.ts` file.

* With the current approach we skip the analysis of the whole attribute if it finds any kind of interpolation, but I assume there are also cases like this: 
 ```html+erb
  <div data-controller="some-controller <%= "some-dynamic-controller" %>">
    <!-- ... -->
  </div>
  ```
  I guess we could ignore this for now and open another issue for it. But I think we should still try to process the analysis for `some-controller` in this case.

What do you think?

**Comment by Gert-JanPeeters** [comment]
I think we should indeed tackle the issues you raised in this issue. No point in postponing it to the future.

Let me work on this a bit more so we also resolve the two issues that you raised 👍🏼

**Review by marcoroth — APPROVED** [review]
This looks great to me @Gert-JanPeeters, thanks! 🙌🏼 

We can merge it if you could resolve the merge conflicts

**Comment by Gert-JanPeeters** [comment]
Ready to be merged 🚀

**Comment by marcoroth** [comment]
Thank you so much @Gert-JanPeeters! 🙌🏼

**Pull request merged**